mmv1/templates/terraform/constants/bigquery_reservation_capacity_commitment.go.erb

```diff
@@ -0,0 +1,6 @@
+func bigqueryReservationCapacityCommitmentPlanDiffSuppressFunc(_, old, new string, _ *schema.ResourceData) bool {
+	if (old == "FLEX" || new == "MONTHLY" || old == "ANNUAL") && new == old+"_FLAT_RATE" {
```

Comment:
Can you help me understand this line (a comment might be helpful as well)? My understanding is: we suppress the diff if the change is `FLEX -> FLEX_FLAT_RATE` or `ANNUAL -> ANNUAL_FLAT_RATE`, and that seems correct. But I think `new == MONTHLY` can never pass the second condition (`MONTHLY != ???_FLAT_RATE`).

Reply:
oops! That's a typo. Nice catch!